content/api/substitutions-reference.apib

```diff
@@ -103,7 +103,7 @@ Expressions return values that are inserted into the template content.  The foll
 {{ airports[code].city }}
```

Comment:
This file has been removed from Master.  Should we add it back?

Reply:
Was this answered elsewhere?